src/components/markdown.js

```diff
@@ -47,16 +47,20 @@ export const newWindowLinkRenderer = (href, title, text) => {
 
 const SimpleMDE = lazy(() => import('react-simplemde-editor'))
 
-export const MarkdownEditor = props => {
-  return h(Suspense, { fallback: centeredSpinner() }, [h(SimpleMDE, _.merge({
-    options: {
-      autofocus: true,
-      renderingConfig: {
-        singleLineBreaks: false
-      },
-      previewClass: ['editor-preview', 'markdown-body'],
-      previewRender: renderAndSanitizeMarkdown,
-      status: false
-    }
-  }, props))])
+/**
+ * N.B. Make sure you memoize options if you're passing any in!
```

Comment:
Nit: Might be a bit clearer to say:
```suggestion
 * N.B. Make sure you memoize 'options' if you're passing any in to 'SimpleMDE'!
```